ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -82,66 +79,19 @@ import {
 } from 'app/utils/stores';
 import { isUsingFreeTierBillingAccount } from 'app/utils/workspace-utils';
 
-const { useState, useEffect, Fragment } = React;
-
-const CostInfo = ({
```

Comment:
I like the extraction of the cost elements into CostPredictor

Reply:
Thanks :-)